pkg/flex/mounter.go

```diff
@@ -319,5 +319,12 @@ func createCRI() (cri.CRI, error) {
 		return cri.NewContainerd("/run/containerd/containerd.sock", "v3io")
 	}
 
+	// NOTE: On some managed kubernetes services, docker is installed but not activated
```

Comment:
on line 317:
// if docker binary exists, create docker. otherwise containerd
change to:
// if docker binary does not exists, use containerd